app/assets/stylesheets/pages/_roasts.scss

```diff
@@ -0,0 +1,255 @@
+$range-track-color: #DBDBDB;
+$range-handle-color-center: #FF6363;
+$range-handle-size: 36px;
+$range-handle-size-big: 67px;
+$range-handle-size-center: 0.45vw;
+
+@mixin flex-centered {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.left-axis {
+  @include fluid-sizeFULL('left', 0px, 140px);
+}
+
+.right-axis {
+  @include fluid-sizeFULL('right', 0px, 140px);
+}
+
+.roasts {
+  display: flex;
+  flex-direction: column;
+  .roasts__slider-wrapper {
+    display: flex;
+    flex-direction: column;
+    order: 2;
+    align-self: center;
+    margin-top: 10%;
+    margin-bottom: 10%;
+    width: 100%;
+    align-items: center;
+    .slider-axis {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 61%;
+      margin-bottom: 3%;
+      @include breakpoint(300px 1000px) {
+        justify-content: space-around;
+        margin-bottom: 4%;
+      }
+      @include breakpoint($large-screens) {
+        width: 100%;
+      }
+      span {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        &.desktop-only {
+          display: none;
+          @include breakpoint($large-screens) {
+            display: flex;
+          }
+        }
+        @include fluid-size('width', 50px, 75px);
+        height: 15px;
+        span {
+          width: 50%;
+          border-right: solid 2.3px $range-track-color;
+        }
+        &.axial-last {
+          border: none;
+        }
+        @include breakpoint($large-screens) {
+          @include fluid-sizeFULL('width', 10px, 120px);
+          justify-content: flex-start;
+        }
+      }
+      img {
+        @include fluid-size('max-width', 21px, 31px);
+        display: flex;
+        margin-left: auto;
+        margin-right: auto;
+
+        @include breakpoint($large-screens) {
+          display: none;
+        }
+      }
+      .axial-text {
+        border: none;
+        display: none;
+        color: $range-track-color;
+        width: initial;
+        height: initial;
+        @include breakpoint($large-screens) {
+          @include fluid-size('font-size', 14px, 28px);
+          display: flex;
+          position: relative;
+        }
+      }
+    }
+    .slider {
+      display: flex;
+      flex-direction: row;
+      width: 64%;
+      @include breakpoint($large-screens) {
+        width: 90%;
+      }
+      img {
+        display: none;
+        align-self: center;
+        margin-left: 1.2vw;
+        margin-right: 1.2vw;
+        @include breakpoint($large-screens) {
+          display: block;
+        }
+      }
+      .slider__range {
+        align-self: center;
+        -webkit-appearance: none;
```

Comment:
_searches furiously to find autoprefixer_

HOW ARE WE NOT USING AUTOPREFIXER? WE GOT TO GET THAT GOING ASAP.